src/dummy/CommandTask.cpp

```diff
@@ -0,0 +1,56 @@
+#include "CommandTask.hpp"
+#include "Arduino.h"
+#include "ArduinoJson.h"
+
+void CommandTask::activity()
+{
+    vTaskDelay(2000); // wait for ip stack!
+
+    struct netconn *conn, *newconn;
+    err_t err;
+
+    /* Create a new connection identifier. */
+    /* Bind connection to well known port number 7. */
```

Comment:
Looks like port 2002 no?

Reply:
oops, copied this comment from somewhere